third_party/gfxreconstruct/framework/decode/file_processor.h

```diff
@@ -288,7 +286,8 @@ class FileProcessor
     virtual bool ProcessStateMarker(const format::BlockHeader& block_header, format::MarkerType marker_type);
     virtual void StoreBlockInfo() {}
 
-    bool run_without_decoders_ = false;
+    bool        run_without_decoders_ = false;
+    std::string absolute_path_;
```

Comment:
why do you prefer to move `absolute_path_` to under // GOOGLE? seems to be a var from LunarG?

Reply:
oh, i see. thanks